<launch>  
  <node pkg="mqtt_ros_bridge" type="imu_bridge.py" name="imu_bridge" output="screen">
      <remap from="/G_Watch_R_5509" to="/imu_data"/>
  </node>
  <node pkg="mqtt_ros_bridge" type="imu_bridge_recordings.py" name="imu_bridge_recording" output="screen">
      <remap from="/G_Watch_R_5509_package" to="/imu_data_package"/>
  </node>
</launch>

